### ISSUE TITLE
Is this repo still maintained? If not, please note in the README file.

### ISSUE THREAD

**Issue by tlkahn** [rustr/rustr#15]

**Comment by CGMossa** [comment]
I would be happy to collaborate on maintaining this project.

**Comment by caiohamamura** [comment]
Yeah, too bad it isn't being maintained. I'd be happy to contribute to it also. Actually this package is really useful and if it went to CRAN as it is, it would be great so everybody could start using it to deploy their own packages. I would be happy to submit it to CRAN, I have some experience doing that.

**Comment by qinwf** [comment]
Yes, I am not maintaining it. Maybe I will go back to this project in the future.

**Comment by CGMossa** [comment]
I'd be more than willing to collaborate and execute on tasks to get this
project modernised. But irregardless, thank you for replying.

**Comment by tjtnew** [comment]
Just highlighting that that it may be worth applying for some funding from the R Consortium for a project like this (https://www.r-consortium.org/blog/2020/03/11/march-2020-isc-call-for-proposals). If not in this round of proposals maybe in a future one.

**Comment by CGMossa** [comment]
Thank you for this suggestion. I have decided to apply for this, as
I truly believe that Rust could help many r-users with their
small-native-code-base needs.

**Comment by andy-thomason** [comment]
I'm torn between extending this project and starting something a bit simpler. In Rust we can be a bit more friendly to R programmers than RCpp has been by using the R API and provide automatic translation of R language objects.

**Comment by CGMossa** [comment]
*I think,* going for safe and sound interfacing between R and Rust is a
very big selling point really. So I'd definitely go for a re-write.
Also, I am sure the internals of R has changed enough that it warrants an
investigation as to if there is something to be gained from a rebuild.

Let me know, if I can assist in any way.

**Comment by andy-thomason** [comment]
There are some pretty good takeaways from this project. The documentation, issues with running embedded R from rust. Most of my daily work is re-writing R code in Rust, so having a 1:1 API with the R API would be nice (ie. as_integer, length, data_frame etc.)

**Comment by andy-thomason** [comment]
I'd love to be able to compile as list of th top 100 functions used in R.

**Comment by andy-thomason** [comment]
If anyone is interested in helping out. I've started something new as "extendr".

**Comment by andy-thomason** [comment]
It has a high level API designed to make translating R code to rust as easy as possible.

https://github.com/extendr/rapi

And a low-level wrapper for libR

https://github.com/extendr/libR-sys

**Comment by qinwf** [comment]
Updated in the readme https://github.com/rustr/rustr/commit/9b7e887070fef7d2af41da618860bf4394ca4622

**Comment by andy-thomason** [comment]
Many thanks, @qinwf . There are many takeaways from this project that are extremely useful to understanding how to do this. You and the others would be very welcome to contribute if you had time, but I suspect that you have other fish to fry.
